django_apscheduler/admin.py

```diff
@@ -27,10 +27,19 @@ def __init__(self, model, admin_site):
         self._django_jobstore = DjangoJobStore()
         self._memory_jobstore = DjangoMemoryJobStore()
 
-        self._jobs_executed = []
+        self._job_queue = []
         self._job_execution_timeout = getattr(
             settings, "APSCHEDULER_RUN_NOW_TIMEOUT", 15
         )
+        self._scheduler = BackgroundScheduler({
+            'apscheduler.executors.default': {
+                'class': 'apscheduler.executors.pool:ThreadPoolExecutor',
+                'max_workers': '1'
+            }
+        })
+        self._scheduler.add_jobstore(self._memory_jobstore)
+        self._scheduler.add_listener(self._handle_execution_event, events.EVENT_JOB_EXECUTED | events.EVENT_JOB_ERROR | events.EVENT_JOB_MISSED | events.EVENT_JOB_MAX_INSTANCES)
```

Comment:
Starting a scheduler as soon as the user views the admin page might be unnecessary - there are no guarantees that they intend to initiate a new job every time that they navigate to this page.